flexget/tests/test_thetvdb.py

```diff
@@ -218,22 +218,25 @@ def test_run():
 
         # Should not expire as it was checked less then an hour ago
         persist['last_check'] = datetime.utcnow() - timedelta(hours=1)
-        with mock.patch(
-            'requests.sessions.Session.request',
-            side_effect=Exception('Tried to expire or lookup, less then an hour since last check'),
-        ):
-            # Ensure series is not marked as expired
-            with Session() as session:
-                mark_expired(session)
-                ep = (
-                    session.query(TVDBEpisode)
-                    .filter(TVDBEpisode.series_id == 73255)
-                    .filter(TVDBEpisode.episode_number == 2)
-                    .filter(TVDBEpisode.season_number == 2)
-                    .first()
-                )
-                assert not ep.expired
-                assert not ep.series.expired
+        with (
+            mock.patch(
+                'requests.sessions.Session.request',
+                side_effect=Exception(
+                    'Tried to expire or lookup, less then an hour since last check'
+                ),
+            )
+            and Session() as session
```

Comment:
Hmm. Shouldn't this be a comma instead of 'and'? It's a bit concerning that the tests still pass. (Unless I'm misunderstanding this.)